app/src/main/java/com/kms/cura/view/activity/DoctorViewActivity.java

```diff
@@ -349,23 +354,52 @@ private void hideProgressDialog() {
 
     @Override
     public void handleEvent(String event, Object data) {
+        int increase = 0;
+        boolean increasement = (boolean) data;
+        if (increasement){
+            increase = 1;
+        }
+        else{
+            increase = -1;
+        }
```

Comment:
Should only do this bit of code if event is either EventConstant.UPDATE_APPT_NOTI_NUMBER or EventConstant.UPDATE_REQUEST_NOTI_NUMBER